keyboards/mwstudio/mw65_rgb/config.h

```diff
@@ -43,19 +43,11 @@
 #define DEBOUNCE 5
 
 #define RGB_DI_PIN B3
-#ifdef RGB_DI_PIN
-    #define RGBLIGHT_SLEEP
-
-    #define RGB_MATRIX_MAXIMUM_BRIGHTNESS 200
-    #define RGB_MATRIX_STARTUP_VAL RGB_MATRIX_MAXIMUM_BRIGHTNESS
-    #define DRIVER_LED_TOTAL 83
-    #define RGBLED_NUM 83
-    #define RGB_MATRIX_KEYPRESSES
-
-    #define RGBLIGHT_HUE_STEP 8
-    #define RGBLIGHT_SAT_STEP 8
-    #define RGBLIGHT_VAL_STEP 8
-#endif
+
+#define RGB_MATRIX_MAXIMUM_BRIGHTNESS 200
+#define RGB_MATRIX_STARTUP_VAL RGB_MATRIX_MAXIMUM_BRIGHTNESS
+#define DRIVER_LED_TOTAL 83
+#define RGB_MATRIX_KEYPRESSES
```

Comment:
just a heads up, for RGB Matrix, you now need to explicitly enable any modes you want to to be used.

Reply:
Just resolved issue, thanks